packages/website/docs/DocSearch-v3.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: DocSearch v3
+authors:
+  - name: Clément Vannicatte
+    title: Software Engineer @ Algolia
+    url: https://github.com/shortcuts
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::info
+
+The following content is for **[DocSearch v3][2]**. If you are using **[DocSearch v2][3]**, see the **[legacy][4]** documentation.
+
+:::
+
+## Introduction
+
+DocSearch v3 is built on top of the latest version of [Algolia Autocomplete][1], which provides better accessibility, increased responsiveness, themability, a better built-in design, and customizability under low-network conditions.
+
+This version has been rewritten in React, and now exposes React components. Vanilla JavaScript version is based on the React version with an alias to Preact.
+
+### Stable version
+
+With the recent release of the stable version of [Algolia Autocomplete][1], and the huge adoption of DocSearch v3, we will start working on a stable release!
+
+Thanks to all the Alpha testers, and to [all the integrations][5] who already supports it!
+
+## Installation
+
+DocSearch packages are available on the [npm][10] registry.
+
+<Tabs
+  groupId="language"
+  defaultValue="js"
+  values={[
+    { label: 'JavaScript', value: 'js', },
+    { label: 'React', value: 'react', }
+  ]
+}>
+<TabItem value="js">
+
+
+```bash
+yarn add @docsearch/js@alpha
+# or
+npm install @docsearch/js@alpha
+```
+
+If you don’t want to use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/js@alpha"></script>
+```
+
+</TabItem>
+<TabItem value="react">
+
+
+```bash
+yarn add @docsearch/react@alpha
+# or
+npm install @docsearch/react@alpha
+```
+
+If you don’t want to use a package manager, you can use a standalone endpoint:
+
+```html
+<script src="https://cdn.jsdelivr.net/npm/@docsearch/react@alpha"></script>
+```
+
+</TabItem>
+
+
+</Tabs>
+
+
+## Get started
+
+<Tabs
+  groupId="language"
+  defaultValue="js"
+  values={[
+    { label: 'JavaScript', value: 'js', },
+    { label: 'React', value: 'react', }
+  ]
+}>
+<TabItem value="js">
+
+
+To get started, you need a [`container`][11] for your DocSearch component to go in. If you don’t have one already, you can insert one into your markup:
+
+```html
+<div id="docsearch"></div>
+```
+
+Then, insert DocSearch into it by calling the [`docsearch`][12] function and providing the container. It can be a [CSS selector][6] or an [Element][7].
+
+Make sure to provide a [`container`][11] (for example, a `div`), not an `input`. DocSearch generates a fully accessible search box for you.
+
+```js app.js
+import docsearch from '@docsearch/js';
+
+import '@docsearch/css';
```

Comment:
We don't say to install `@docsearch/css` above.

Reply:
It's because it's a dependency of the JS and React packages, should we at least mention that it is?